submissions/karelskiy/HTML and CSS practice/index.html

```diff
@@ -7,87 +7,117 @@
-            <div id="fig">
-                <a href="#"><figure>
-                    <img src="img/telegram-app.png">
-                    <figcaption>Telegram</figcaption>
-                </figure></a>
-                <a href="#"><figure>
-                    <img src="img/snapchat.png">
-                    <figcaption>Snapchat</figcaption>
-                </figure></a>
-               <a href="#"> <figure>
-                    <img src="img/skype.png">
-                    <figcaption>Skype</figcaption>
-                </figure></a>
-                <a href="#"><figure>
-                    <img src="img/msg.png">
-                    <figcaption>MSG</figcaption>
-                </figure></a>
-                <a href="#"><figure>
-                    <img src="img/whatsapp.png">
-                    <figcaption>WhatsApp</figcaption>
-                </figure></a>
-                <a href="#"><figure>
-                    <img src="img/discord-logo.png">
-                    <figcaption>Discord</figcaption>
-                </figure></a>
-                <a href="#"> <figure>
-                    <img src="img/facebook-new.png">
-                    <figcaption>Facebook</figcaption>
-                </figure></a>
-                <a href="#"><figure>
-                    <img src="img/instagram.png">
-                    <figcaption>Instagram</figcaption>
-                </figure></a>
-                <a href="#"> <figure>
-                    <img src="img/youtube.png">
-                    <figcaption>YouTube</figcaption>
-                </figure></a>
-                <a href="#"><figure>
-                    <img src="img/crunchyroll.png">
-                    <figcaption>Crunchyroll</figcaption>
-                </figure></a>
-                <a href="#"> <figure>
-                    <img src="img/linkedin.png">
-                    <figcaption>LinkedIn</figcaption>
-                </figure></a>
-                <a href="#" id="more"><span>More</span></a>              
-                <a href="#" class="hidden"><figure>
-                    <img src="img/line-me.png">
-                    <figcaption>Line</figcaption>
-                </figure></a>
-                <a href="#" class="hidden"> <figure>
-                    <img src="img/flickr.png">
-                    <figcaption>Flickr</figcaption>
-                </figure></a>
-                <a href="#" class="hidden"> <figure>
-                    <img src="img/xing.png">
-                    <figcaption>Xing</figcaption>
-                </figure></a>
-                <a href="#" class="hidden"><figure>
-                    <img src="img/pinterest.png">
-                    <figcaption>Pinterest</figcaption>
-                </figure></a>
+    <nav class="nav">
+        <a href="#" class="lImg li" tabindex="1">Images</a>
+        <a href="#" class="lMail li" tabindex="2">Mail</a>
+        <a href="#" class="lDoe li"><img class="img-Doe" src="img/JhonDoe.jpg"
```

Comment:
All interactive elements have focus state in the browser by default. Using `tabindex` you redefine default browser order of focused elements. Be careful redefining the default focus order.

Reply:
As i understand correctly, i need remove this tabindex?

Reply:
I this case you don't need `tabindex`
[Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) about `tabindex` why and how to use it.

Reply:
Or, i should add in all elements?

Reply:
OK, thanks.